crypto/ec/ec_pmeth.c

```diff
@@ -36,6 +36,8 @@ typedef struct {
     size_t kdf_ukmlen;
     /* KDF output length */
     size_t kdf_outlen;
+	/* ECDSA K (nonce) type */
+    char nonce_type;
```

Comment:
Why is this signed?

Reply:
kdf_type defined is this struct is also a (signed) char
Is it necessary to change it here?

Reply:
Unless it goes negative it should be unsigned.